### DISCUSSION TITLE
Is there a way to add filters / keyword blacklist to suggestion / translation

### DISCUSSION THREAD

**Discussion by Heziode** [WeblateOrg/weblate#5527] [Q&A]
# Problem

Some spammers add suggestion to my Weblate instance. Messages contains keywords or links.

# Question

Is there a way to automatically refuse suggestion/translation that contains by giving a blacklist of keywords or regex ?

**Comment by nijel** [comment]
You can use Akismet service for fighting with spam, see https://docs.weblate.org/en/latest/admin/optionals.html#spam-protection

**Comment by Heziode** [comment]
I am not sure to understand where I have to install it.

I running weblate inside a Docker container, I have to intsall Akismet inside this container or on the host ?

**Reply by nijel** [reply]
The module is already installed there, all you need is configuring it, see https://docs.weblate.org/en/latest/admin/install/docker.html#envvar-WEBLATE_AKISMET_API_KEY

I've updated the docs to cover this in a4faeb2ff860153facc137d39f96775f0998a7d4.
